app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -21,3 +21,10 @@ end
 child :host_collections => :host_collections do
   attributes :id, :name
 end
+
+attributes :facts
+
+# All of the following requires another API call to fetch from Candlepin
+attributes :installedProducts
```

Comment:
can you change this to installed_products, sorry i didn't catch this earlier.  Old systems api used camel case for some dumb reason (i guess because candlepin does)